### ISSUE TITLE
[feat] any plans to update wix v5 from wix v3?

### ISSUE THREAD

**Issue by reiify** [tauri-apps/tauri#10348]
### Describe the problem

I installed WiXToolset v5 because it was written on the official site that support for this (wix v3) version would end soon, so I downloaded the latest one.

### Describe the solution you'd like

I tried to integrate Tauri with WiX v5 using a PowerShell command, but it couldn't find my application (because I can't compile it). For compilation, Tauri uses the candle.exe and light.exe commands, which are no longer supported. Instead, the wix build command shoul be used.

### Additional context

Sorry for my bad english, I'm translated it from russian.

**Comment by FabianLars** [comment]
Yes there are plans, but it's not a priority at the moment and it's not a trivial change either, especially with wix' new distribution mode.

So yeah, don't wait for it unless you want to contribute to Tauri yourself, but _we_ won't work on it until after the v2 stable release.

**Comment by reiify** [comment]
Thanks for fast answer.

**Comment by FabianLars** [comment]
Actually, let's keep this open so the request is properly tracked :)

**Comment by elibroftw** [comment]
Any update or guidelines? I recently used wix v7 for work, so I might be able to help out given some guidance.

**Comment by FabianLars** [comment]
Nope, didn't want to deal with the distribution changes and the Maintenance Fee thing kinda killed it for me. Not a priority atm but if you could give us a rundown of changes you think are required to adapt (mainly concerned about the nugget shit, xml changes are to be expected), that would help already.

We'll focus on tauri v3 (for gtk4) for a bit so i assume this can't get much attention until afterwards.
